container/example/example.tf

```diff
@@ -72,10 +88,30 @@ module "target" {
     port     = "80"
   }]
 
-  tags {
-    terraform   = "True"
-    environment = "dev"
+  tags = "${var.tags}"
+}
+
+# Create a target group with listeners.
+module "targetTCP" {
+  source = "github.com/TeliaSoneraNorge/divx-terraform-modules//container/target"
+
+  prefix            = "${var.prefix}"
+  vpc_id            = "${module.vpc.vpc_id}"
+  load_balancer_arn = "${module.nlb.arn}"
+
+  target {
+    protocol        = "TCP"
+    port            = "9000"
+    health_protocol = "TCP"
```

Comment:
Seems this and health_port should be removed. Have you tested `health` with NLB?

Reply:
I tested that the code ran without complaint - but I did not add 9000 to the lb security group and didn't have a service behind that was expecting a TCP connection on port 9000